zepid/calc/utils.py

```diff
@@ -1,42 +1,89 @@
 import warnings
-import math
+from typing import NamedTuple
+
+import numpy as np
 from scipy.stats import norm
 
 
-def risk_ci(events, total, alpha=0.05, confint='wald'):
-    """Calculate two-sided (1-alpha)% Confidence interval of Risk. Note
-    relies on the Central Limit Theorem, so there must be at least 5 events
-    and 5 nonevents
+class Results(NamedTuple):
+    """
+    A simple wrapper class for the results of marginal estimates
+    """
+    point_estimate: float
```

Comment:
How do you feel about these names? Any you want to change?

Reply:
Names look good to me, but what are your thoughts on using ``measure`` instead of ``name``? I am not opposed to ``name`` since there is not really a prior API for this

Reply:
I like measure a lot more 👍